test/testvm.py

```diff
@@ -36,6 +36,33 @@
 
 SSH_WARNING = re.compile(r'Warning: Permanently added .* to the list of known hosts.*\n')
 
+# NOTE: This must be kept in sync with vm-prep!
+#
+ip_from_mac = {
+    '52:54:00:9e:00:00': '10.111.111.10',
+    '52:54:00:9e:00:01': '10.111.111.11',
+    '52:54:00:9e:00:02': '10.111.111.12',
+    '52:54:00:9e:00:03': '10.111.111.13',
+    '52:54:00:9e:00:04': '10.111.111.14',
+    '52:54:00:9e:00:05': '10.111.111.15',
+    '52:54:00:9e:00:06': '10.111.111.16',
+    '52:54:00:9e:00:07': '10.111.111.17',
+    '52:54:00:9e:00:08': '10.111.111.18',
+    '52:54:00:9e:00:09': '10.111.111.19',
+    '52:54:00:9e:00:0a': '10.111.111.20',
+    '52:54:00:9e:00:0b': '10.111.111.21',
+    '52:54:00:9e:00:0c': '10.111.111.22',
+    '52:54:00:9e:00:0d': '10.111.111.23',
+    '52:54:00:9e:00:0e': '10.111.111.24',
+    '52:54:00:9e:00:0f': '10.111.111.25',
+
+    '52:54:00:9e:00:F0': '10.111.111.100',
+    '52:54:00:9e:00:F1': '10.111.111.101',
+    '52:54:00:9e:00:F2': '10.111.111.102',
+    '52:54:00:9e:00:F3': '10.111.111.103',
+    '52:54:00:9e:00:F4': '10.111.111.104'
```

Comment:
I suggest we don't duplicate this information. Ideally we would ask dnsmasq about it. But if we can't we should at least keep it in one place.